README.md

```diff
@@ -197,13 +197,93 @@ final String resultMessage = result.getChoices().get(0).getMessage().getContent(
 
 See [an example in our Spring Boot application](e2e-test-app/src/main/java/com/sap/ai/sdk/app/controllers/OpenAiController.java)
 
-### Chat completion with a model not in defined `OpenAiModel`
+### Chat completion with a model not defined in `OpenAiModel`
 
 ```java
 final OpenAiChatCompletionOutput result =
     OpenAiClient.forModel(new OpenAiModel("model")).chatCompletion(request);
 ```
 
+### Stream chat completion
+
```

Comment:
(minor) I think it would be helpful to describe in 2 sentences what we are documenting here: passing the stream to a frontend.

Maybe it would even be worthwhile to first showcase using the stream on it's own, e.g. simply printing to stdout, and only in a second step show how one can return it to a frontend non-blocking. Probably easier to understand that way 😉